web-ui/src/pages/SelfReviewsPage.jsx

```diff
@@ -1,57 +1,61 @@
-import React, {useContext, useCallback, useEffect, useState} from "react";
```

Comment:
Nearly all the changes in this file are from Prettier formatting. I'll add comments below that describe the real changes.